site/en/meet-the-team/index.njk

```diff
@@ -0,0 +1,10 @@
+---
+layout: 'layouts/events.njk'
```

Comment:
We try to avoid having .njk files in language directories, because this cannot be picked up by our translation service. Is there a reason to not have the code directly in events.njk instead of includes here?

Reply:
No problem - have switched to an MD file and moved the includes to the layout. Just FYI I asked about i18n and apparently the events are going up in English only. Understand it's best to future-proof regardless, of course!